ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1761,3 +1761,33 @@ FCU_DFVSC_RTM_CO2M_HWZTC_CHWZTC:
   - CO2M
   - HWZTC
   - CHWZTC
+
+
+FCU_DXZTC_DFSS:
+  guid: "5688519e-f88e-4620-8bd5-89d91467028d"
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
+  - DXZTC
+  - DFSS
+  opt_uses:
+  - failed_alarm
+  - filter_alarm
+
+FCU_RHDHS_DXDC_DFSS_RTM:
+  guid: "f60a2563-da8f-4360-b2d9-f471c065356c"
+  description: "FCU with temperature & pressure monitor, humidity monitor and water leak & status alarm."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DXDC
+  - DFSS
+  - RTM
+  opt_uses:
+  - water_leak_alarm
+  - failed_alarm
+  - high_discharge_air_static_pressure_alarm
+  - low_discharge_air_static_pressure_alarm
+  - low_discharge_air_flowrate_alarm
```

Comment:
Please use existing type `DFM` and remove this field

Reply:
understood ma'am, thanks.